book/src/specification/textual-representation/concrete-syntax.md

```diff
@@ -17,22 +17,23 @@ process.
 
 ## Items
 
-### Alias definitions
+### Constant definitions
 
-Alias definitions are used to give names to terms that can be later used in
+Constant definitions are used to give names to terms that can be later used in
 other places in the binary description. For example:
 
 ```fathom
 Byte = U8;
```

Comment:
There are some constant definitions like this in the book that we'll have to update. Sadly the book is not covered by the tests yet.

Reply:
Ah yes, another good catch – i remember you mentioning this and being unsure of when it'd become relevant!